ui/app/src/components/Footer.tsx

```diff
@@ -40,17 +40,17 @@ export default function Footer(): JSX.Element {
             </a>
           </li>
           <li>
-            {loading ? (
+            {isLoading ? (
               <CircularProgress size="1rem" />
-            ) : response && response.version.length > 0 ? (
-              response.version
+            ) : data !== undefined && data.version !== '' ? (
```

Comment:
yeah usually I'm using `data.version.length > 0` because I found it's more accurate to say `a non empty string` and also because I'm a bit worried about the empty space condition (as a miss type)